SmartDeviceLink/SDLStreamingVideoLifecycleManager.m

```diff
@@ -619,7 +651,10 @@ - (void)sdl_hmiStatusDidChange:(SDLRPCNotificationNotification *)notification {
     }
 
     // if startWithProtocol has not been called yet, abort here
-    if (!self.protocol) { return; }
+    if (self.protocol == nil) {
+        SDLLogV(@"No session established with head unit. HMI status is not relevant.");
```

Comment:
```suggestion
        SDLLogW(@"No session established with head unit. HMI status is not relevant.");
```

Reply:
I'm seeing a failure here on Sync 3.0 if I unplug / replug a video streaming app. When I cancel / try to bring the app back, it fails here with no protocol.

Reply:
Fixed first comment.

Reply:
I haven't seen this issue. If you see it while testing can you note the `hmiLevel` of the SDL app and whether or not the device app is foregrounded or backgrounded?